core/players-handbook-2024/classes/class-ranger.xml

```diff
@@ -0,0 +1,419 @@
+			<p>You invoke spirits of nature to magically hide yourself. As a Bonus Action, you can give yourself the Invisible condition until the end of your next turn.</p>
+			<p class="indent">You can use this feature a number of times equal to your Wisdom modifier (minimum of once), and you regain all expended uses when you finish a Long Rest.</p>
+		</description>
+		<sheet alt="Nature's Veil" action="Bonus Action" usage="{{natveil:usage}}/Long Rest">
+			<description>You can give yourself the Invisible condition until the end of your next turn.</description>
+		</sheet>
+		<rules>
+			<stat name="natveil:usage" value="1" bonus="base" />
+			<stat name="natveil:usage" value="wisdom:modifier" bonus="base" />
+		</rules>
+	</element>
+	<element name="Level 17: Precise Hunter" type="Class Feature" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_CLASS_FEATURE_RANGER_PRECISE_HUNTER">
+		<requirements>!ID_INTERNAL_FEATURE_REPLACEMENT_RANGER_24_PRECISE_HUNTER</requirements>
+		<description>
+			<p>You have Advantage on attack rolls against the creature currently marked by your <i>Hunter’s Mark.</i></p>
+		</description>
+		<sheet alt="Precise Hunter">
+			<description>You have Advantage on attack rolls against the creature currently marked by your Hunter’s Mark.</description>
+		</sheet>
+	</element>
+	<element name="Level 18: Feral Senses" type="Class Feature" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_CLASS_FEATURE_RANGER_FERAL_SENSES">
+		<requirements>!ID_INTERNAL_FEATURE_REPLACEMENT_RANGER_24_FERAL_SENSES</requirements>
+		<description>
+			<p>Your connection to the forces of nature grants you Blindsight with a range of 30 feet.</p>
+		</description>
+		<sheet alt="Feral Senses">
+			<description>You have Blindsight with a range of 30ft.</description>
+		</sheet>
+		<rules>
+			<grant type="Vision" id="ID_VISION_BLINDSIGHT" />
+			<stat name="blindsight:range" value="30" bonus="base" />
+		</rules>
+	</element>
+	<element name="Level 19: Epic Boon" type="Class Feature" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_CLASS_FEATURE_RANGER_EPIC_BOON">
+		<requirements>!ID_INTERNAL_FEATURE_REPLACEMENT_RANGER_24_EPIC_BOON</requirements>
+		<description>
+			<p>You gain an Epic Boon feat (see chapter 5) or another feat of your choice for which you qualify. Boon of Dimensional Travel is recommended.</p>
+		</description>
+		<sheet display="false"/>
+		<rules> <!-- to be changed 
+			<select type="Feat" name="Epic Boon (Ranger 19)" supports="   , 19"  /> -->
+		</rules>
+	</element>
+	<element name="Level 20: Foe Slayer" type="Class Feature" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_CLASS_FEATURE_RANGER_FOE_SLAYER">
+		<requirements>!ID_INTERNAL_FEATURE_REPLACEMENT_RANGER_24_FOE_SLAYER</requirements>
+		<description>
+			<p>The damage die of your Hunter's Mark is a dlO rather than a d6.</p>
+		</description>
+		<sheet alt="Foe Slayer">
+			<description>The damage die of your Hunter's Mark is a dlO rather than a d6.</description>
```

Comment:
```suggestion
			<p>The damage die of your Hunter's Mark is a d10 rather than a d6.</p>
		</description>
		<sheet alt="Foe Slayer">
			<description>The damage die of your Hunter's Mark is a d10 rather than a d6.</description>
```